libyul/AsmScopeFiller.cpp

```diff
@@ -153,12 +138,11 @@ bool ScopeFiller::operator()(Block const& _block)
 	// an entry in the scope according to their visibility.
 	for (auto const& s: _block.statements)
 		if (s.type() == typeid(FunctionDefinition))
-			if (!boost::apply_visitor(*this, s))
+			if (!registerFunction(bost::get<FunctionDefinition>(s)))
```

Comment:
Typo. Should be `boost::get<FunctionDefinition>`?

Reply:
Yes, but I think there is another problem.